server/config/topic_page/sdg/sdg.textproto

```diff
@@ -0,0 +1,13794 @@
+  metadata {
+    topic_id: "sdg"
+    topic_name: "SDG"
+  }
+  categories {
+    title: "01: No Poverty"
```

Comment:
Does this need to be 01 to get it sorted, or will the order be decided by the ordering here?

If order can be chosen, can we remove "0" before 1?  Natalie noted how that is a bit odd.

Reply:
Ok, ignore comments about titles.  I think those should come from the KG given the number of SVs involved